aws-datastore/src/test/java/com/amplifyframework/datastore/appsync/AppSyncClientTest.java

```diff
@@ -88,10 +90,12 @@ public void validateBaseSyncQueryGen() throws JSONException, DataStoreException
             ) -> {
                 try {
                     GraphQLRequest<PaginatedResult<ModelWithMetadata<BlogOwner>>> request =
-                            endpoint.buildSyncRequest(BlogOwner.class, null, null);
+                            endpoint.buildSyncRequest(ModelSchema.fromModelClass(BlogOwner.class), null, null);
                     endpoint.sync(request, onResult, onError);
                 } catch (DataStoreException datastoreException) {
                     onError.accept(datastoreException);
+                } catch (AmplifyException amplifyException) {
+                    // Todo failed converting Model Class to ModelSchema
```

Comment:
Instead of catching this, can you just have the test throw an `AmplifyException`, to ensure the test fails for this scenario?

Reply:
Yes, will do.